src/fix/rgbfix.1

```diff
@@ -179,6 +183,46 @@ sans global checksum:
 .Pp
 .D1 $ rgbfix -cjsv -k A4 -l 0x33 -m 0x1B -p 0xFF -r 3 -t SURVIVALKIDAVKE \
 SurvivalKids.gbc
+.Sh TPP1
+TPP1 is a new homebrew mapper, designed for easier handling and a larger ROM cap, while being somewhat backwards compatible with traditional MBCs.
```

Comment:
```suggestion
TPP1 is a homebrew mapper designed as a functional superset of the common traditional MBCs, allowing larger ROM and RAM sizes combined with other hardware features.
```

Reply:
Doesn't it have some incompatibilities? I must be misremembering, then.

Reply:
I copied the language from the spec intro. It also says "it does not intend to be compatible with previous mappers".